share/mdnjs/parse.py

```diff
@@ -214,6 +216,18 @@ def writerows(self):
           'disambiguation': disambig
         })
       for mdn in self.inverted_index[keyword]:
+        # add redirect for Web/Api pages
+        if any( word in mdn.title for word in self.CLASS_WORDS ) and len(mdn.title.split('.')) > 1:
```

Comment:
What do you mean?  I'm just trying to avoid creating a redirect for cases where the only word in the title is also in the class_words.